P/PETSc/build_tarballs.jl

```diff
@@ -326,6 +361,8 @@ ENV["MPITRAMPOLINE_DELAY_INIT"] = "1"
 # Build the tarballs.
 # NOTE: llvm16 seems to have an issue with PETSc 3.18.x as on apple architectures it doesn't know how to create dynamic libraries  
 build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
-               augment_platform_block, julia_compat="1.10", 
-               preferred_gcc_version = v"9",  
-               preferred_llvm_version=v"13")
+               augment_platform_block, 
+               julia_compat="1.6", 
```

Comment:
You want `julia_compat` to be `1.9`, since we are using LBT 5.8 in this build (which is the right thing to do).